types/insane/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for insane 1.0.0
+
+type AllowedSchemes = "http" | "https" | "mailto";
+
+type AllowedTags =
+    | "a"
+    | "abbr"
+    | "article"
+    | "b"
+    | "blockquote"
+    | "br"
+    | "caption"
+    | "code"
+    | "del"
+    | "details"
+    | "div"
+    | "em"
+    | "h1"
+    | "h2"
+    | "h3"
+    | "h4"
+    | "h5"
+    | "h6"
+    | "hr"
+    | "i"
+    | "img"
+    | "ins"
+    | "kbd"
+    | "li"
+    | "main"
+    | "ol"
+    | "p"
+    | "pre"
+    | "section"
+    | "span"
+    | "strong"
+    | "sub"
+    | "summary"
+    | "sup"
+    | "table"
+    | "tbody"
+    | "td"
+    | "th"
+    | "thead"
+    | "tr"
+    | "u"
+    | "ul";
+
+type EmptyObject = Record<string, never>;
+
+type AllowedAttributes = Record<AllowedTags, string[] | never[]>;
+
+type AllowedClasses = Record<AllowedTags, string[]> | EmptyObject;
+
+type Filter = (token: string) => string;
```

Comment:
The README demonstrates returning a boolean. Seems like `unknown` is the right return type here.

Reply:
Done. It expects a boolean based on the tests